test/unit/driver/test_digitalocean.py

```diff
@@ -0,0 +1,230 @@
+#  Copyright (c) 2015-2018 Cisco Systems, Inc.
```

Comment:
Also note on changing to `... Red Hat ...`.

Reply:
Included in lastest commit 👍